app/assets/stylesheets/vat-repayment-tracker.css

```diff
@@ -296,6 +297,10 @@ details summary:focus,
     text-decoration: none;
 }
 
+.js-enabled .govuk-tabs__tab:link, .js-enabled .govuk-tabs__tab:visited, .js-enabled .govuk-tabs__tab:hover, .js-enabled .govuk-tabs__tab:active, .js-enabled .govuk-tabs__tab:focus {
```

Comment:
Wasn't this file meant for the copied CSS for the general purpose WCAG contrast/colour changes and to be kept separate from the application specific css tweaks?

It looks like this PR now mixes in application specific CSS tweaks. This will make it difficult to maintain in the future if new WCAG colour/contrast stuff comes in; we'd need to then disentangle the old copied WCAG stuff from the application specific bits added here. I'd suggest it's best to keep the files separate and ideally have the files named so that we avoid confusion between the copied general WCAG stuff and the application specific tweaks.

Reply:
Or is this just an update of the general WCAG colour/contrast stuff and not application specific?